reference/features/sorting.md

```diff
@@ -0,0 +1,200 @@
+# Sorting
+
+By default, MeiliSearch focuses on ordering results according to their relevancy. You can alter this sorting behavior so users can decide at search time what type of results they want to see first.
+
+This can be useful in many situations, such as when a user wants to see the cheapest products available in a webshop.
+
+::: tip
+Sorting at search time can be particularly effective when combined with [placeholder searches](/learn/what_is_meilisearch/features.md#placeholder-search).
+:::
+
+## Configuring MeiliSearch for sorting at search time
+
+To allow your users to sort results at search time you must:
+
+1. Decide which attributes you want to use for sorting
+2. Add those attributes to the `sortableAttributes` index setting
+3. Update MeiliSearch's [ranking rules](/learn/core_concepts/relevancy.md) (optional)
+
+### Select attributes for sorting
+
+MeiliSearch allows you to sort results based on document fields. Only fields containing numbers or strings can be used for sorting.
```

Comment:
Fields containing arrays also work. If I understood correctly if you have two different documents: 

```json
{
   "id": 1,
   "something": [4, 5]
},
{
   "id": 2,
   "something": [3, 6]
}
```

and you search on `something:asc`, then document with id `1` will be more relevant as the lowest number in both array is `3`.
@gmourier Could you confirm?

Reply:
This is exact! Only the first level is taken into account.